guides/git-workflow.md

```diff
@@ -1,21 +1,89 @@
+
+## Workflow Instructions <a name="workflow-instructions"></a>
+
+Below we provide step-by-step instructions on how to employ each workflow.
+
+### Opening a GitHub Issue <a name="github-issue"></a>
+Regardless of which workflow you chose, before contributing, you should determine if the eventual change is significant enough to warrant a GitHub issue; if you think that this change will be solving a significant problem or providing a significant enhancement to the project, then it would be advantageous to open an issue under the `spacetelescope` repository. This will allow both contributors and repository maintainers to keep track of the project and capture any needed context of the particular change. Any appropriate individuals should be assigned to the issue, and any appropriate label(s) should be tagged.
+
+### The Branching Workflow <a name="branching-workflow"></a>
+
+1. Make a local copy of the `spacetelescope` repository by cloning the repository (e.g. `git clone https://github.com/username/repository_name.git`).  Note that, unless you explicitly delete your clone of the repository, this only has to be done once.
+
+2. Ensure that your clone is pointing to the `spacetelescope` remote repository with `git remote -v`.  You should see that `origin` corresponds to the URL from step (1).  If it doesn't, you can add it with `git remote add origin https://github.com/username/repository_name.git`.
+
+3. Create a branch on the clone to develop software changes on.  Branch names should be short but descriptive (e.g. `new-database-table` or `fix-ingest-algorithm`), and not too generic (e.g. `big-fix`).  Consistent use of hyphens is encouraged.
+    1. `git branch <branchname>`
+    2. `git checkout <branchname>` - you can use this command to switch back and forth between existing branches.
+    3. Perform local software changes using the nominal `git add`/`git commit -m` cycle:
+       1. `git status` -  allows you to see which files have changed.
+       2. `git add <new or changed files you want to commit>`
+       3. `git commit -m 'Explanation of changes you've done with these files'`
+
+4. Push the branch to the `spacetelescope` repository with `git push origin <branchname>`.
+
+5. In the `spacetelescope` repository, create a pull request for the recently pushed branch.  You will want to set the "base" option to point to the branch you would like to merge changes into (e.g. `master`), and the "compare" option to point to the new branch (i.e. `<branchname>`).  Note that if the branch is still under heavy development, you can put `WIP:` at the beginning of the pull request title to signify that the pull request is still a work in progress (i.e. `WIP: Example Pull Request Title`).  Not until the `WIP:` tag is explicitly removed will the pull request be deemed 'mergeable'.
```

Comment:
This bit about 'WIP' should be removed and replaced instead with reference to using the Draft PR function.

Reply:
Great suggestion